src/poset/poset.go

```diff
@@ -1355,6 +1400,12 @@ func (p *Poset) GetFrame(roundReceived int64) (Frame, error) {
 	//Every participant needs a Root in the Frame. For the participants that
 	//have no Events in this Frame, we create a Root from their last consensus
 	//Event, or their last known Root
+	// INFO: Needs PeerSet from dynamic_participants
+	//peerSet, err := p.Store.GetPeerSet(roundReceived)
```

Comment:
do we need this?

Reply:
In the current state it works fine with it commented. 
This is just a place holder. I am currently working on a major update, dynamic participants, where this is required.